dimagi/utils/couch/migration.py

```diff
@@ -135,10 +136,10 @@ def _migration_get_couch_model_class(cls):
         raise NotImplementedError()
 
     def _migration_get_couch_object(self):
-        if not self.couch_id:
+        if not getattr(self, self.couch_id_name, None):
```

Comment:
i don't have preference for method vs class attr if it only lives on the orm model class, but if it were to also be on the couch class then i'd prefer a method to avoid saving unnecessary data